yang-module-versioning/draft-ietf-netmod-yang-module-versioning.xml

```diff
@@ -1168,336 +685,252 @@ module ietf-yang-library-revisions {
+        <t>The YANG module specified in this document defines a schema for data that is designed to be accessed via network management protocols such as NETCONF <xref target="RFC6241"/> or RESTCONF <xref target="RFC8040"/>.  The lowest NETCONF layer is the secure transport layer, and the mandatory-to-implement secure transport is Secure Shell (SSH) <xref target="RFC6242"/>.  The lowest RESTCONF layer is HTTPS, and the mandatory-to-implement secure transport is TLS
+          <xref target="RFC8446"/>.</t>
+        <t>The NETCONF access control model <xref target="RFC8341"/> provides the means to restrict access for particular NETCONF or RESTCONF users to a preconfigured subset of all available NETCONF or RESTCONF protocol operations and content.</t>
+        <t>This document does not define any new protocol or data nodes that are writable.</t>
+        <t>This document updates YANG Library <xref target="RFC8525"/> with augmentations to include two boolean leafs that indicate whether status deprecated and status obsolete schema nodes are implemented by the server. These read-only augmentations do not add any new security considerations beyond those already present in <xref target="RFC8525"/>.</t>
+      </section>
+    </section>
+    <section anchor="iana" title="IANA Considerations">
+      <section anchor="yang-module-registrations" title="YANG Module Registrations">
+        <t>This document requests IANA to registers a URI in the "IETF XML Registry" <xref target="RFC3688"/>. Following the format in RFC 3688, the following registrations are requested.
+          <?rfc subcompact="yes" ?>
+          <list style="empty">
+            <t>URI: urn:ietf:params:xml:ns:yang:ietf-yang-revisions</t>
+            <t>Registrant Contact: The IESG.</t>
+            <t>XML: N/A, the requested URI is an XML namespace.</t>
+          </list>
+          <list style="empty">
+            <t>URI: urn:ietf:params:xml:ns:yang:ietf-yang-status-conformance</t>
+            <t>Registrant Contact: The IESG.</t>
+            <t>XML: N/A, the requested URI is an XML namespace.</t>
+          </list>
+          <?rfc subcompact="no" ?>
+        </t>
+        <t>The following YANG module is requested to be registred in the "IANA Module Names" <xref target="RFC6020"/>.  Following the format in RFC 6020, the following registrations are requested:</t>
+        <t>The ietf-yang-revisions module:</t>
+        <t>
+          <list>
+            <t>Name: ietf-yang-revisions</t>
+            <t>XML Namespace: urn:ietf:params:xml:ns:yang:ietf-yang-revisions</t>
+            <t>Prefix: rev</t>
+            <t>Reference: [RFCXXXX]</t>
+          </list>
+        </t>
+        <t>The ietf-yang-status-conformance module:</t>
+        <t>
+          <list>
+            <t>Name: ietf-yang-status-conformance</t>
+            <t>XML Namespace: urn:ietf:params:xml:ns:yang:ietf-yang-status-conformance</t>
+            <t>Prefix: yl-rev</t>
```

Comment:
This isn't right, is it?  Shouldn't this be yl-conf?

Reply:
Thanks, ys-conf.